lib/install.js

```diff
@@ -218,7 +208,11 @@ function install (gyp, argv, callback) {
         })
 
         // start unzipping and untaring
-        req.pipe(gunzip).pipe(extracter)
+        res.pipe(tar.extract({
+          strip: 1,
+          cwd: devDir,
+          filter: isValid
+        }).on('close', _ => setTimeout(afterTarball, 1000)).on('error', cb))
```

Comment:
Can you use a non-arrow function here?  We still nominally support v0.10 and v0.12.  Does tar@3 support pre-ES6 node?

Side question: why the 1s delay?